pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -361,6 +384,52 @@ private boolean forceLog(BrokerResponse brokerResponse, long totalTimeMs) {
     return false;
   }
 
+
+  private void scoreResponse(BrokerResponse response, RequestStatistics stats) {
+
+    try {
+
+      if (response == BrokerResponseNative.EMPTY_RESULT || response == BrokerResponseNative.NO_TABLE_RESULT
```

Comment:
No-table is the result of the user sending query to a non-existent table? Do you want to count that as an error response?

Reply:
Non-existent table issue is caught earlier with the auth check, so I think this should be OK. This can also happen when we have a valid table, but EV doesn't reflect it. A signal on the second case seems important than a false alarm on the first. What do you think?